azkaban-common/src/main/java/azkaban/utils/OsMemoryUtil.java

```diff
@@ -0,0 +1,110 @@
+package azkaban.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Utility class for getting system memory information
+ */
+class OsMemoryUtil {
```

Comment:
That's cool. There is no mandate as such. 
There are examples like `sun.security.x509.IPAddressName` but there are counter examples like `com.sun.jmx.snmp.SnmpIpAddress` in Java itself.